charts/proxy/templates/deployment.yaml

```diff
@@ -23,11 +23,11 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         livenessProbe:
           httpGet:
-            path: /healthz
```

Comment:
this should be accessible 
https://github.com/gomods/athens/blob/12ce2a6499bd9fe3b0486968008497f0fb03f9ee/cmd/proxy/actions/app_proxy.go#L25

Reply:
@michalpristas My bad. The helm chart currently references v0.2.0 in its default configuration where the health checks were only bound to `/`. Would setting the default value to `canary` be an option here?

Reply:
@zerok good catch. would it be possible to add an `if` on the version in here, so that either way (`v0.2.0` and `canary` - no need to look for other versions than that) this health check can reference the right path?

Reply:
@arschles I've put the paths used for these checks behind a template 🙂